views/bootswatch.pug

```diff
@@ -13,15 +13,15 @@ block content
                         - if (i === helpers.theme.selected(config, theme))
                             button.btn.btn-primary(type="button", disabled) Active!
                         - else
-                            button.btn.btn-primary(type="button", onclick="tryIt(" + i + ")") Try it!
+                            button.btn.btn-primary(type="button", data-theme=i) Try it!
```

Comment:
@jmervine: I'm thinking, we should be able to do this without JS via Pug. Like doing `a href=/?theme=x`.